src/Microsoft.Health.Dicom.Api/Features/BackgroundServices/StartBlobMigrationService.cs

```diff
@@ -50,9 +52,9 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                     var operationsClient = scope.ServiceProvider.GetRequiredService<IDicomOperationsClient>();
 
                     // We also need to ensure if the operation client already not completed
-                    if (operationsClient != null && !await operationsClient.IsBlobCopyCompletedAsync(stoppingToken))
+                    if (operationsClient != null && !await operationsClient.IsBlobCopyCompletedAsync(_operationId, stoppingToken))
```

Comment:
This can never be `null` as `GetRequiredService` throws if the service cannot be resolved.